pages/login_page.rb

```diff
@@ -1,31 +1,40 @@
 require_relative "./page.rb"
-
 # would deal with anny assertions, logging, etc
 
 
 
 class LoginPage < Page
     # UI MAPPING
-    USERNAME_FIELD = "CSS SELECTOR";
-    PASSWORD_FIELD = "DSFADSF";
-
-    def initialize url, driver
+    USERNAME_FIELD = { :name => "email"}
+    PASSWORD_FIELD = { :name => "password"}
+    LOGIN_BUTTON =   { :name => "button"}
+    LOGIN_PATH = "/login"
```

Comment:
login path would probably be better left at root since `${url_path}/login` can cause oddities if we ar hecking anything that requires logging in and out

Reply:
For myself, this was addressed. Will be in next push